test/Main.hs

```diff
@@ -133,3 +196,53 @@ traces ::
 traces ll comp expected = case evalTerm (Tracing ll DoTracing) comp of
   Left err -> assertFailure $ "Did not compile: " <> Text.unpack err
   Right (_, _, logs) -> assertEqual "" expected logs
+
+isTrue ::
+  (forall (s :: S). Term s PBool) ->
+  IO ()
+isTrue comp = case evalTerm NoTracing comp of
+  Left err -> assertFailure $ "Did not compile: " <> Text.unpack err
+  Right (res, _, _) -> case res of
+    Left err -> assertFailure $ "Execution errored: " <> show err
+    Right t -> unless (plift t) (assertFailure "is false")
```

Comment:
Instead of these, you can use `plift'` which gives you error directly

Reply:
same thing for below

Reply:
I need to throw an `AssertionError` specifically for `tasty-hunit` to work here.